deploy/027_deploy_scroll_spokepool.ts

```diff
@@ -0,0 +1,26 @@
+import { L2_ADDRESS_MAP } from "./consts";
+import { deployNewProxy, getSpokePoolDeploymentInfo } from "../utils/utils.hre";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getChainId } = hre;
+  const { hubPool } = await getSpokePoolDeploymentInfo(hre);
+  const chainId = parseInt(await getChainId());
+
+  // Initialize deposit counter to very high number of deposits to avoid duplicate deposit ID's
+  // with deprecated spoke pool.
+  // Set hub pool as cross domain admin since it delegatecalls the Adapter logic.
+  const initArgs = [
+    L2_ADDRESS_MAP[chainId].scrollERC20GatewayRouter,
+    L2_ADDRESS_MAP[chainId].scrollMessenger,
+    1_000_000,
+    hubPool.address,
+    hubPool.address,
+  ];
+  const constructorArgs = [L2_ADDRESS_MAP[chainId].l2Weth, 3600, 32400];
```

Comment:
what are these values? Please add comments

Reply:
Done: https://github.com/across-protocol/contracts-v2/pull/381/commits/f3fba07303819bafdb12f5acf4f0d7f552f6f2c2

These are identical to the other spoke pool deployments so I've opened a PR for this here: https://github.com/across-protocol/contracts-v2/pull/389